spec/features/decks_index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+  
+describe "list of decks page tests" do
+  let(:user) { FactoryGirl.create :user, email: "unique@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.